app/views/components/gds/exportsInputTextWithPrefix.scala.html

```diff
@@ -60,14 +74,10 @@ <h1 class="govuk-label-wrapper">
  </h1>
  }
 
- @hintKey.map { hint =>
-  <span id=@{s"${field.id}-hint"} class="govuk-hint">
-   @messages(hint)
-  </span>
- }
+ @hint(hintKey.map(messages(_)), prefixHintKey.map(messages(_)))
 
  @allErrors.map { error => <span id="error-message-@{field.name}-input" class="govuk-error-message">@error</span> }
 
- <span id=@{s"${field.id}-before"} class="@{s"govuk-body $prefixClass"}" aria-label="@prefixLabelKey.map(messages(_)).getOrElse("")">@messages(prefixKey)</span>
- <input class="@inputClasses" value="@field.value" id=@field.id name=@field.id type="text" aria-describedby=@{s"${field.id}-hint"}>
+ <span id=@{s"${field.id}-before"} aria-hidden="true" class="@{s"govuk-body $prefixClass"}">@messages(prefixKey)</span>
```

Comment:
Removing aria-label from this span fixes the original issue.  It prevents the screen-reader tabbing to the input field "prefix" span.